react-components/src/hooks/use3dScenes.tsx

```diff
@@ -23,8 +25,16 @@ export type ExternalId = string;
 
 export type SceneData = {
   name: string;
+  cameraTranslationX: number;
+  cameraTranslationY: number;
+  cameraTranslationZ: number;
+  cameraEulerRotationX: number;
+  cameraEulerRotationY: number;
+  cameraEulerRotationZ: number;
   cadModelOptions: AddReveal3DModelOptions[];
   image360CollectionOptions: AddImageCollection360DatamodelsOptions[];
+  groundPlanes: GroundPlaneProperties[];
```

Comment:
Do we support multiple ground planes?

Reply:
Yes, we support multiple groundplanes. This could be useful if you e.g. have sea level and subsea level